deploy/examples/operator-openshift.yaml

```diff
@@ -570,6 +570,13 @@ data:
   # NOTE: the value here serves as an example and needs to be
   # updated with node labels that define domains of interest
   # CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # Disable read affinity for rbd volumes.
+  # CSI_ENABLE_READ_AFFINITY: "false"
+  # CRUSH location labels define which node labels to use
+  # as CRUSH location. This corresponds to the values used in
+  # ceph CRUSH map.
+  # CSI_CRUSH_LOCATION_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
```

Comment:
>This will be a very edge case. Makes sense to keep it, but how about the following to keep the default simple?
They should only need to add additional custom labels. By default rook would automatically tell CSI about all the supported OSD topology labels and the hostname label.

There maybe chances that k8s topology labels leads to choosing a crush location that is not optimal even when the user has supplied additional labels.

I'd prefer to provide an option to provide completely new set of labels.

Reply:
Ok if a new set of labels is needed, then the default (blank) should be that rook will tell CSI about all the built-in labels. If they specify anything, then it would all be reset to their labels.

Reply:
yes